README.md

```diff
@@ -16,7 +16,9 @@ Or install directly gazebo8 from http://gazebosim.org/tutorials?tut=install_ubun
 libgazebo7-dev or libgazebo8-dev must be installed.
 
 ````
-git clone the repo
+sudo apt-get install libgazebo7-dev
+sudo apt-get install libgazebo8-dev
```

Comment:
We can't have both installed ... or at least with ROS. I will merge and add a OR ! ^^